website/docs/d/kubernetes_cluster.html.markdown

```diff
@@ -37,6 +37,10 @@ The following attributes are exported:
 
 * `id` - The ID of the Kubernetes Managed Cluster.
 
+* `api_server_authorized_ip_ranges` - The IP ranges to whitelist for incoming traffic to the masters.
+
+-> **NOTE:** `api_server_authorized_ip_ranges` Is currently in Preview on an opt-in basis. To use it, enable feature `APIServerSecurityPreview` for `namespace Microsoft.ContainerService`. For an example of how to enable a Preview feature, please visit [How to enable the Azure Firewall Public Preview](https://docs.microsoft.com/en-us/azure/firewall/public-preview)
```

Comment:
this is a (read-only) Data Source, so this probably doesn't need the disclaimer here:

```suggestion
```

Reply:
Resolved in new commit

Reply:
mind double-checking that? it's still showing the original commit here?